recipes/runtimeqml/all/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 3.21.1)
+
+project(runtimeqml LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 17)
+
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+
+file(GLOB HEADERS "${CMAKE_CURRENT_SOURCE_DIR}/*.h" "${CMAKE_CURRENT_SOURCE_DIR}/*.hpp")
```

Comment:
Given there's 4 files I think we could use a list, wDYT?

Reply:
Yes, I hate GLOB and it's not recommended by CMake devs.